guides/common/modules/con_using-network-boot-to-provision-hosts.adoc

```diff
@@ -0,0 +1,80 @@
+[id="using-network-boot-to-provision-hosts"]
+= Using network boot to provision hosts
+
+You can provision bare-metal instances with {Project} by using one of the following methods:
```

Comment:
This sentence does not match the heading IMHO. Could you please reword either slightly?

Reply:
I will do that kind of things in a follow-up PR. Now I'm only moving things around.